x/bridgingfee/ibc_module.go

```diff
@@ -0,0 +1,119 @@
+package bridgingfee
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	transferkeeper "github.com/cosmos/ibc-go/v6/modules/apps/transfer/keeper"
+	channeltypes "github.com/cosmos/ibc-go/v6/modules/core/04-channel/types"
+	porttypes "github.com/cosmos/ibc-go/v6/modules/core/05-port/types"
+	"github.com/cosmos/ibc-go/v6/modules/core/exported"
+	delayedackkeeper "github.com/dymensionxyz/dymension/v3/x/delayedack/keeper"
+	rollappkeeper "github.com/dymensionxyz/dymension/v3/x/rollapp/keeper"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+const (
+	ModuleName = "bridging_fee"
+)
+
+// IBCModule is responsible for charging a bridging fee on transfers coming from rollapps
+// The actual charge happens on the packet finalization
+// based on ADR: https://www.notion.so/dymension/ADR-x-Bridging-Fee-Middleware-7ba8c191373f43ce81782fc759913299?pvs=4
+type IBCModule struct {
+	porttypes.IBCModule
```

Comment:
It was explictily set to `transfer.IBCModule` as it's a wrapper.
To enforce no middlewares between

Reply:
fixed sorry